backend/api/Services/InspectionService.cs

```diff
@@ -96,5 +109,44 @@ private IQueryable<Inspection> GetInspections(bool readOnly = true)
             inspection = await Update(inspection);
             return inspection;
         }
+
+        public async Task<IDAInspectionDataResponse?> GetInspectionStorageInfo(string inspectionId)
+        {
+            string relativePath = $"InspectionData/{inspectionId}/inspection-data-storage-location";
+
+            var response = await idaApi.CallApiForUserAsync(
+                ServiceName,
+                options =>
+                {
+                    options.HttpMethod = HttpMethod.Get.Method;
+                    options.RelativePath = relativePath;
+                }
+            );
+
+
+            if (response.StatusCode == HttpStatusCode.Accepted)
+            {
+                logger.LogInformation("Inspection data storage location for inspection with Id {inspectionId} is not yet available", inspectionId);
+                return null;
+            }
+
+            if (response.StatusCode == HttpStatusCode.InternalServerError)
+            {
+                logger.LogError("Inetrnal server error when trying to get inspection data for inspection with Id {inspectionId}", inspectionId);
+                return null;
+            }
+
+            if (response.StatusCode == HttpStatusCode.NotFound)
+            {
+                logger.LogError("Could not find inspection data for inspection with Id {inspectionId}", inspectionId);
+                return null;
+            }
```

Comment:
Add a fourth if statement, checking that we have the right StatusCode

if (response.StatusCode != HttpStatusCode.Success)
    // log error here as well